### PR TITLE
New package: NordicSemiconductor.nrfutil version 7.6.0

### PR DESCRIPTION
**Pull request by kenr**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/119722)

### CODE REVIEW

**Comment by stephengillie** [comment]
Hi @kenr,

This package has a dependency on the `VCRedist` package:
image

File: manifests/n/NordicSemiconductor/nrfutil/7.6.0/NordicSemiconductor.nrfutil.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.5.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: NordicSemiconductor.nrfutil
+PackageVersion: 7.6.0
+InstallerType: portable
+Commands:
+- nrfutil
```
Comment (stephengillie) [inline manifests/n/NordicSemiconductor/nrfutil/7.6.0/NordicSemiconductor.nrfutil.installer.yaml:8]:
```suggestion
- nrfutil
Dependencies:
  PackageDependencies:
  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

Comment (kenr) [reply manifests/n/NordicSemiconductor/nrfutil/7.6.0/NordicSemiconductor.nrfutil.installer.yaml:8]: Thanks @stephengillie, hopefully it goes through now 👍

**Comment by stephengillie** [comment]
Hi @kenr,

This package has passed validation, and we'll be accepting packages with dependencies I believe with our 1.6 release.

**Comment by kenr** [comment]
Hi, not sure how to proceed on this one. Pinging @ImJoakim to try to reach out to a moderator.
Please let me know if the was the wrong way of doing this.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**